client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Submit a Work page
![Screenshot of Submit a Work page showing 14px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/9f692261-4336-4fda-b47f-336ccb9e729f)

Reply:
These now appear as 16px.